rules/network/initial_access_unsecure_elasticsearch_node.toml

```
@@ -38,9 +38,10 @@ timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-(event.dataset: network_traffic.http OR (event.category: network_traffic AND network.protocol: http)) AND
-    status:OK AND destination.port:9200 AND network.direction:inbound AND NOT http.response.headers.content-type:"image/x-icon" AND NOT
-    _exists_:http.request.headers.authorization
+(event.dataset:network_traffic.http or (event.category:network_traffic and network.protocol:http)) and
+    http.response.status_phrase:ok and destination.port:9200 and network.direction:inbound and
+    not http.response.headers.content-type:"image/x-icon" and
```

we can resolve this header by switching to mime_type field
```suggestion
    not http.response.mime_type:"image/x-icon" and
```